README.rst

```diff
@@ -24,24 +24,27 @@ We use Git tags to pin a stack version and release a stack image to Docker hub.
 
 Versioning is done in the following manner:
 
-- major stack versions are marked with ``<scrapy version>[-py3]`` tag.
+- major stack versions are marked with ``<scrapy version>[-pyX]`` tag.
 
-  Note: Lack of ``-py3`` suffix means that a stack is built using Python 2.
+  **Note**: All tags prior to ``2.0`` used Python 2 as its default version.
+  Support for Python 2 was dropped on version ``2.0``, so from that version onwards all tags base on Python 3
+  The presence of a ``-py3`` suffix on an older tag means that specific stack uses updated Python 3 interpreter.
```

Comment:
I like this require a lot.

I would suggest using ``.. note::``, but given this is the README, and https://github.com/github/markup/issues/1256, I think this is better.

Reply:
A lot of... changes? Any suggestions on what to change?
I agree some more emphasis would be better, but as you mention, that is not yet available.

Reply:
Sorry, I meant this *rewrite. Nothing to change here, just saying that I like this change :slightly_smiling_face:

Reply:
Great, I'll tag this a resolved, then.